modules/tenant-management/src/Volo.Abp.TenantManagement.Blazor/Pages/TenantManagement/TenantManagement.razor.cs

```diff
@@ -67,18 +71,21 @@ protected virtual Task CloseEditConnectionStringModal()
 
         protected virtual async Task UpdateConnectionStringAsync()
         {
-            await CheckPolicyAsync(ManageConnectionStringsPolicyName);
-
-            if (TenantInfo.UseSharedDatabase || TenantInfo.DefaultConnectionString.IsNullOrWhiteSpace())
+            if (ManageConnectionStringValidations.ValidateAll())
             {
-                await AppService.DeleteDefaultConnectionStringAsync(TenantInfo.Id);
+                await CheckPolicyAsync(ManageConnectionStringsPolicyName);
+
+                if (TenantInfo.UseSharedDatabase || TenantInfo.DefaultConnectionString.IsNullOrWhiteSpace())
```

Comment:
You define as `[Required]` for `TenantInfoModel.DefaultConnectionString`, but here we control is that null or empty.